src/app/zones/locales/en-us/index.yaml

```diff
@@ -92,6 +88,8 @@ zones:
               enabled: {zoneIngressEnabled}
           egress:
               enabled: {zoneEgressEnabled}
```

Comment:
**Suggestion**: While we’re here, would you mind fixing these, too? They’re not technically broken but still indented one level too far.

```suggestion
            enabled: {zoneIngressEnabled}
          egress:
            enabled: {zoneEgressEnabled}
```